core/src/main/scala/com/rackspace/com/papi/components/checker/Validator.scala

```diff
@@ -98,8 +107,8 @@ class Validator private (private val _name : String, val startStep : Step, val c
 
 
   private val platformMBeanServer = ManagementFactory.getPlatformMBeanServer()
-  private val objectName = new ObjectName("\"com.rackspace.com.papi.components.checker\":type=\"Validator\",scope=\""+
-                                 name+"\",name=\"checker\"")
+  private val objectName = new ObjectName("com.rackspace.com.papi.components.checker:type=Validator,scope="+
+                                 name+",name=checker")
```

Comment:
The quotes were there because that's what the metrics library did with the rest of its object...otherwise you won't find all the metrics in the same place ...I'll check to see if it's still doing that.  If it is, you probably want to add them back.